portfolio/src/main/java/com/google/sps/data/Email.java

```diff
@@ -0,0 +1,17 @@
+package com.google.sps.data;
+
+/** Email. */
+public final class Email {
+
+  private final long id;
+  private final String emailaddress;
+  private final String emailmessage;
+  private final long timestamp;
+
+  public Email(long id, String emailaddress, String emailmessage, long timestamp) {
```

Comment:
This class doesn't expose any methods to read the contents of the email instance after creation. Consider adding getter methods for accessing the fields.